docs/configurable-menu.md

```diff
@@ -4,18 +4,22 @@ When developing an app with uPortal-app-framework, you can use the configurable
 The main menu is opened by the hamburger icon in the top left corner of the app. On medium and large screens, the menu includes only the content you've
 configured. On mobile, the menu includes a top bar with portal-wide components like the notifications bell, username menu, and new features button.
 
-If you don't configure the menu, the hamburger icon and side navigation will only be displayed on small screens.
+If you don't configure the menu, the hamburger icon and side navigation will not be displayed, and the portal-wide content will be displayed in the top bar on small screens.
 
 Basic configuration occurs in your app's `override.js` file. See the [configuration doc](configuration.md) for an example.
 
+**Important:** Side navigation depends on the `<frame-page>` directive. Use the `<frame-page>` directive as the outermost containing element for your app view(s). See the [frame-page.html](https://github.com/uPortal-Project/uportal-app-framework/blob/master/components/portal/misc/partials/frame-page.html) file to understand how this is constructed.
+
 ## How to use
 
-There are two levels of complexity you can use when configuring your menu.
+There are two levels of complexity you can use when configuring the contents of your menu.
+
+1. [Simple](#simple-configuration): Menu options that have a text label, a url, and an optional material icon (i.e. basic links)
+2. [Custom template](#custom-menu-template): Varied menu options, some of which may be tied to an angular controller and affect functionality of the app (e.g. a theme-switching toggle)
 
-1. Simple: Menu options that have a text label, a url, and an optional material icon (i.e. basic links)
-2. Complex: Varied menu options, some of which may be tied to an angular controller and affect functionality of the app (e.g. a theme-switching toggle)
+You can also try the _**experimental**_ [push-content feature](#push-content-navigation) if you want the side navigation to be open upon page load and appear as part of the page content. The side nav will push content when it is opened and give up the space when closed.
```

Comment:
"Experimental" is interesting. If it's just "experimental", then maybe this doesn't have to be a major version bump? If we suspect we're going to break downstream apps tho, I'd favor the major version bump to signal this.

Reply:
I mainly meant that as a "take caution using this" because it may be difficult to work with (due to the way it affect on-page content). I'm open to saying "caution" instead of "experimental" if you think that would be better. I do intend for this to be a permanent change.